taxi/forms.py

```diff
@@ -0,0 +1,80 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django.core.validators import MinLengthValidator, MaxLengthValidator
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",)
+
+    license_number = forms.CharField(
+        required=True,
+        validators=[MinLengthValidator(8), MaxLengthValidator(8)]
+    )
+
+    def clean_license_number(self):
```

Comment:
It's ok, but validating the `license_number` directly in the **model** is a more centralized and efficient approach.
It ensures consistency across all entry points (forms, admin, APIs) and reduces code duplication.
Consider moving your validation logic to the **model** to adhere to the **DRY** principle and improve maintainability.

Reply:
I don't agree with you because, People from other country can have other template for license number (for example it cab be more than 8 characters or it can be all letters or all can be number, letter of upper letter).
if you will be able to change it from admin panel it will better.